### DISCUSSION TITLE
commits aren't recorded in my contributions calendar

### DISCUSSION THREAD

**Discussion by hfanzaou** [community/community#61698] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

commits aren't recorded in my contributions calendar 
i tried different solution i find out on github but nothing works

**Comment by radar-blip** [comment]
In your settings, 
make sure you have the bottom box checked and _update_ preferences.

![contributions](https://github.com/community/community/assets/39196767/40df4f31-82dd-4a18-b08c-49ba8d203597)

If that doesn't help, here is another link to other possible solutions: [Contributions that are counted](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted)
